ui/store/actions.ts

```diff
@@ -3511,10 +3511,13 @@ export function resolvePendingApproval(
     await submitRequestToBackground('resolvePendingApproval', [id, value]);
     // Before closing the current window, check if any additional confirmations
     // are added as a result of this confirmation being accepted
+
+    ///: BEGIN:ONLY_INCLUDE_IN(build-main,build-beta,build-flask)
```

Comment:
won't this cause the lint to fail on mmi build? (lint after codefence)

Reply:
Those will be taken cared in a next PR 👍🏼